industrial_trajectory_filters/CMakeLists.txt

```diff
@@ -2,6 +2,8 @@ cmake_minimum_required(VERSION 2.8.3)
 
 project(industrial_trajectory_filters)
 
+add_compile_options(-std=c++11)
```

Comment:
Based on my own understanding of things (and [this](http://stackoverflow.com/questions/23995019/what-is-the-modern-method-for-setting-general-compile-flags-in-cmake) random SO post to back me up), wouldn't `target_compile_options(..)` be a better way of doing this?